feature-ui-solve-problem/src/main/kotlin/team/duckie/app/android/feature/ui/solve/problem/screen/QuizScreen.kt

```diff
@@ -148,6 +148,19 @@ private fun ContentSection(
     pagerState: PagerState,
     state: SolveProblemState,
 ) {
+    val keyboardController = LocalSoftwareKeyboardController.current
+    val focusRequester = remember { FocusRequester() }
+    val currentProblem = state.problems[pagerState.currentPage].problem
+
+    LaunchedEffect(pagerState.currentPageOffsetFraction) {
```

Comment:
```suggestion
    val isCurrentPageOffsetFractionZero = remember { 
        derivedStateOf{ pagerState.currentPageOffsetFraction == 0f }
    }
    
    LaunchedEffect(isCurrentPageOffsetFractionZero)
```
이렇게 하면 currentPageOffsetFraction이 zero일 때에만 블럭이 실행될 수 있을 것 같네요!

Reply:
derivedStateOf를 활용할 수 있었네요.. 오.. 감사합니다 반영했습니다.